src/bootstrap/datatable.html

```diff
@@ -54,43 +54,42 @@
       <tr if.bind="loadingIndicator && loading">
         <td colspan.bind="colspan" innerhtml.bind="loadingIndicator"></td>
       </tr>
+      
+      <tr if.bind="!loading" containerless repeat.for="row of data">
```

Comment:
http://aurelia.io/docs/api/templating/function/containerless/

Now it will render without the `tr`.
Please remove the `containerless`.